README.md

```diff
@@ -36,7 +36,30 @@ class MyWidget extends StatelessWidget {
 
 View the Flutter app in the `example` directory to see all the available `FontAwesomeIcons`.
 
+## Include pro icons
```

Comment:
Instead of this, why not just make the user add the Pro fonts as normal fonts? It would avoid having to use a custom package, and could be as simple as a couple lines in the `pubspec.yaml`

Reply:
Ah I see, the `IconData` requires the package name. Maybe we could work around that...